### PR TITLE
ci(refs T31847): add workflow for pushing to ADO

### PR DESCRIPTION
**Pull request by shumyachkin**
this workflow can be used to push changes to ADO

### CODE REVIEW

File: .github/workflows/ado_sync.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+jobs:
+  push_to_ado:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Push to ADO
+        run: |
+          git remote add ADO https://$(echo $ADO_TOKEN)@$ADO_URI/$ADO_REPO.git
```
Comment (mussbach) [inline .github/workflows/ado_sync.yml:27]: Does this work?

**Review by mussbach — COMMENTED** [review]
one question, otherwise lgtm

File: .github/workflows/ado_sync.yml (hunk 1 above)
Comment (shumyachkin) [reply .github/workflows/ado_sync.yml]: this attempt didn't work so i changed it to http authorization

**Review by mussbach — APPROVED** [review]
lgtm

**Pull request merged**